pages/en/lb2/Model-definition-JSON-file.md

```diff
@@ -580,20 +583,23 @@ Author will have personal information such as social security number etc, and th
 the author of the book will not get those information back (from [GitHub](https://github.com/strongloop/loopback-datasource-juggler/pull/400) pull request).
 The property value is an array of strings, and each string in the array must match a property name defined for the model.
 
-An example of a hidden property is User.email:
+An example of a protected property is User.email:
 
 {% include code-caption.html content="common/models/user.json" %}
 ```javascript
-...
+{
+  ...
   "properties": {
     ...
     "email": {
       "type": "string",
       "required": true
     },
-...
-   "protected": ["email"],
-...
+    ...
+  },
+  "protected": ["email"],
```

Comment:
>  well actually user.json has not protected property : should we fix this in loopback or change the example?

I don't fully understand the purpose of `protected` properties. @raymondfeng what's your opinion?

If I had to decide, I'd rather fix the example. E.g.

```diff
- An example of a protected property is User.email:
+ An example configuring `email` as a protected property:
```